rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -419,39 +373,29 @@ void main_window::InstallPkg(const QString& dropPath)
 		// Run PKG unpacking asynchronously
 		scope_thread worker("PKG Installer", [&]
 		{
-			if (pkg_install(pkg_f, local_path + '/', progress, path))
+			if (pkg_install(path, progress))
 			{
 				progress = 1.;
 				return;
 			}
 
-			// TODO: Ask user to delete files on cancellation/failure?
 			progress = -1.;
 		});
+
 		// Wait for the completion
 		while (std::this_thread::sleep_for(5ms), std::abs(progress) < 1.)
 		{
 			if (pdlg.wasCanceled())
 			{
 				progress -= 1.;
-#ifdef _WIN32
-				taskbar_progress->hide();
```

Comment:
these changes cause the taskbarbutton to remain filled/active after installation

Reply:
![image](https://user-images.githubusercontent.com/23019877/33183127-e4ef7cd8-d076-11e7-816f-c452d2c03658.png)